podder_task_foundation/utilities/data_file_loader.py

```diff
@@ -35,14 +35,14 @@ def load(self, path: Path) -> Union[Dict, List]:
             raise UnsupportedFileFormatError(path)
         data = None
         if file_format == "json":
-            data = json.loads(path.read_text(), object_pairs_hook=OrderedDict)
```

Comment:
should add encoding optional parameter in load method also

Reply:
I also add encoding optional here.
However, I don't think we need to add an encoding option in `json.loads` method. 
Thanks.